apps/demos/src/side-nav.component.scss

```diff
@@ -0,0 +1,83 @@
+// redeclare variables since the barista build cannot deal
+// with a relative reference to the lib folder here
+$gray-200: #e6e6e6;
+$gray-300: #cccccc;
+$turquoise-600: #00a1b2;
+
+.dt-demos-side-nav-searchform {
+  padding: 8px;
+}
+
+// replace this with the sidenav component as soon as this is ready
+.dt-demos-side-nav {
+  background-color: #f2f2f2;
+  font-family: 'BerninaSansWeb';
```

Comment:
can we add OpenSans as falback?

Reply:
Not going to touch the actual code in this PR if not needed